apps/frontend/src/libs/helpers/avatar-helper.ts

```diff
@@ -0,0 +1,11 @@
+const ZERO = 0;
+
+function getAvatarInitials(name: null | string | undefined): string {
```

Comment:
this helper is related to the `users` module so put it there

use an arrow function instead